storm-server/src/main/java/org/apache/storm/scheduler/resource/ResourceAwareScheduler.java

```diff
@@ -115,6 +122,7 @@ public void schedule(Topologies topologies, Cluster cluster) {
                 scheduleTopology(td, cluster, submitter, orderedTopologies);
             }
         }
+        evictedTopologiesMap = tmpEvictedTopologiesMap;
```

Comment:
The idea of making `evictedTopologiesMap` a result after a complete schedule round is a good idea. 

But the code here makes `evictedTopologiesMap` and `tmpEvictedTopologiesMap` the same object.  
So tmpEvictedTopologiesMap.clear == evictedTopologiesMap.clear. This won't achieve the goal here.

Reply:
Fixed on above comments.